app/src/main/AndroidManifest.xml

```diff
@@ -102,7 +102,36 @@
         <activity
             android:name=".ImportExportActivity"
             android:label="@string/importExport"
-            android:theme="@style/AppTheme.NoActionBar" />
+            android:exported="true"
+            android:theme="@style/AppTheme.NoActionBar">
+
+            <!-- ZIP Intent Filter -->
+            <intent-filter >
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:mimeType= "application/zip" />
+                <data android:scheme="content"/>
+                <data android:host="*"/>
+            </intent-filter>
+
+            <!-- JSON Intent Filter -->
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:mimeType="application/json" />
+                <data android:scheme="content"/>
+                <data android:host="*"/>
+            </intent-filter>
+
+            <!-- CSV Intent Filter -->
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:scheme="content"/>
+                <data android:host="*" />
+                <data android:mimeType="text/comma-separated-values" />
+            </intent-filter>
```

Comment:
Could you give the filters labels? Right now when I click a file Catima just says "Import/Export" when the only thing you can do in this flow is import. Maybe "Import cards" would be the best option

Reply:
Sure, will do.